packages/file_selector/file_selector/README.md

```diff
@@ -1,5 +1,7 @@
 # file_selector
 
+<?code-excerpt path-base="excerpts/packages/file_selector_example"?>
```

Comment:
What is this `path-base` pointing to?

Reply:
It's where code-exceprts is generating the output when its run. It's a combination of our tooling's call to generate them (which makes it a relative `excerpts` directory) and how the excerpter itself generates output directories.